SeleniumWebDriverBasics/SeleniumWebDriverBasics/WebObjects/EmailPage.cs

```diff
@@ -0,0 +1,106 @@
+        private static readonly BaseElement subjectField = new BaseElement(By.XPath("//input[@class='composeTextField ComposeSubject-TextField']"));
+        private static readonly BaseElement bodyField = new BaseElement(By.XPath("//div[contains(@class, 'cke_enable_context_menu')]"));
+        private static readonly BaseElement draftsOption = new BaseElement(By.XPath("//a[@data-title='Drafts']"));
+        private static readonly BaseElement draftEmail = new BaseElement(By.XPath("//span[@class='mail-MessageSnippet-FromText']"));
+        private static readonly BaseElement sendButton = new BaseElement(By.XPath("(//button[contains(@class, 'ComposeControlPanelButton-Button')])[1]"));
+        private static readonly BaseElement popupWindow = new BaseElement(By.XPath("//a[@class='control link link_theme_normal ComposeDoneScreen-Link']"));
+        private static readonly BaseElement userLogo = new BaseElement(By.XPath("(//div/img[@class='user-pic__image'])[1]"));
+        private static readonly BaseElement logOutOption = new BaseElement(By.XPath("//span[text()='Log out']"));
+        private static readonly BaseElement deleteButton = new BaseElement(By.XPath("//span[text()='Delete']"));
+        private static readonly BaseElement deletedOption = new BaseElement(By.XPath("//a[@data-title='Deleted']"));
+        private static readonly BaseElement allEmailsCheckbox = new BaseElement(By.XPath("(//span[@class='checkbox_view'])[1]"));
+        private static readonly BaseElement actualUserName = new BaseElement(By.XPath("//a[contains(@class, 'user-account_left-name')]/span[1]"));
+        private static readonly BaseElement actualAdressee = new BaseElement(By.XPath("(//span[contains(@class, 'mail-MessageSnippet-Item_sender')]/span)[1]"));
+        private static readonly BaseElement actualSubject = new BaseElement(By.XPath("(//span[contains(@class, 'mail-MessageSnippet-Item_subject')]/span)[1]"));
+        private static readonly BaseElement actualMessageBody = new BaseElement(By.XPath("(//span[contains(@class, 'mail-MessageSnippet-Item_firstline')]/span)[1]"));
+        private static readonly BaseElement actualDraftInfoMessage = new BaseElement(By.XPath("//span[text()='No messages in Drafts']"));
+        private static readonly BaseElement actualTrashInfoMessage = new BaseElement(By.XPath("//span[text()='No messages in Trash']"));
+
+        public void SwitchToEmailPage()
+        {
+            var currentTab = Browser.driver.WindowHandles.Last();
+            Browser.driver.SwitchTo().Window(currentTab);
+        }
+
+        public void CreateDraftEmail(string subjectText, string bodyText)
+        {
+            createEmailButton.Click();
+            recepientField.Click();
+            recepientOption.Click();
+            subjectField.SendKeys(subjectText);
+            bodyField.SendKeys(bodyText);
+            draftsOption.Click();
+        }
+
+        public void CreateAndSendEmail(string subjectText, string bodyText)
```

Comment:
One method should perform one action. This method repeats CreateDraftEmail() partially. Please remove unnecessary steps.